ringpop.go

```diff
@@ -304,16 +314,10 @@ func (rp *Ringpop) Uptime() (time.Duration, error) {
 	return time.Now().Sub(rp.startTime), nil
 }
 
-func (rp *Ringpop) emit(event interface{}) {
-	for _, listener := range rp.listeners {
-		go listener.HandleEvent(event)
-	}
-}
-
-// RegisterListener adds a listener to the ringpop. The listener's HandleEvent method
-// should be thread safe.
+// RegisterListener is DEPRECATED, use AddListener
```

Comment:
Can you help me understand the call sites for which we have to call this function? Can ringpop applications subscribe to ring events?